charts/tfy-agent/templates/resource-quota.yaml

```diff
@@ -4,9 +4,9 @@ kind: ResourceQuota
 metadata:
   name: {{ include "tfy-agent.fullname" . }}-critical-pods
   labels:
-    {{- include "resource-quotas.labels" . | nindent 4 }}
+    {{- include "resource-quotas.commonLabels" . | nindent 4 }}
```

Comment:
this is incorrect - it should be either `tfy-agent.commonLabels` or `tfy-agent.resourceQuotaAnnotations` or in whichever way you plant to name the helper function